mindjunkies/courses/forms.py

```
@@ -3,6 +3,7 @@
 
 from .models import Course, CourseInfo, CourseToken, Rating
 
+text_area = "form-input mt-1 block w-full p-2 border border-gray-300 rounded-md focus:ring-indigo-500 focus:border-indigo-500"
```

Ruff (0.8.2)

Line too long (126 > 119)

(E501)